### ISSUE TITLE
Only admins can create communities?

### ISSUE THREAD

**Issue by newnetmp3** [discuitnet/discuit#6]
I have gotten the site up & running and have been able to make an admin user and a normal user account, however no matter what i set 'forumCreationReqPoints:' to my normal user is unable to create a community.

`defaultFeedSort: hot
disableForumCreation: false
forumCreationReqPoints: 2
maxForumsPerUser: 15
imagesFolderPath: "images"`

**Comment by previnder** [comment]
Yeah, right now, normal users can't create communities. There are a couple of security issues to fix there before I can enable that.

**Comment by TylerCode** [comment]
I assume this is at least partially related to the issue #2? Maybe this can help push that conversation one way or another too.
